superset/assets/javascripts/utils/common.js

```diff
@@ -86,3 +86,12 @@ export function getShortUrl(longUrl, callback) {
     },
   });
 }
+
+export function isTruthy(obj) {
```

Comment:
acknowledging I have very little context here- this seems like a bad road to go down. We use YAML for plenty of stuff over here and there have been many issues caused by someone not expecting `True`/`t`/`yes` to be true...

Reply:
It has to do with backward compatibility here. There's been a few instances of `CheckboxControl`  becoming a `SelectControl`. In this case the `show_brush` used to be checkbox and now is `auto`, `Yes` or 'No'.

Since it's not the first time and won't be the last, we decided to go this route. 

The alternatives would be to either:
* write database migrations to alter all slices and replace `True` with `'Yes'` but that's more work and db migration have maintenance overhead.
* systematically use `if (fd.show_brush === true || fd.show_brush === 'yes')` everywhere